spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -522,10 +522,483 @@
+          end
+        end
+
+        it 'logs the rate limited analytics event for doc_auth' do
+          attach_and_submit_images
+          expect(fake_analytics).to have_logged_event(
+            'Rate Limit Reached',
+            limiter_type: :idv_doc_auth,
+          )
+        end
+
+        context 'successfully processes image on last attempt' do
+          before { DocAuth::Mock::DocAuthMockClient.reset! }
+
+          it 'proceeds to the next page with valid info' do
+            expect(page).to have_current_path(idv_document_capture_url)
+            expect(page).not_to have_content(t('doc_auth.headings.document_capture_selfie'))
+            attach_and_submit_images
+            expect(page).to have_current_path(idv_ssn_url)
+
+            visit idv_document_capture_path
+
+            expect(page).to have_current_path(idv_session_errors_rate_limited_path)
+          end
+        end
+      end
+
+      it 'catches network connection errors on post_front_image', allow_browser_log: true do
+        DocAuth::Mock::DocAuthMockClient.mock_response!(
+          method: :post_front_image,
+          response: DocAuth::Response.new(
+            success: false,
+            errors: { network: I18n.t('doc_auth.errors.general.network_error') },
+          ),
+        )
+
+        attach_and_submit_images
+
+        expect(page).to have_current_path(idv_document_capture_url)
+        expect(page).to have_content(I18n.t('doc_auth.errors.general.network_error'))
+      end
+
+      it 'does not track state if state tracking is disabled' do
+        allow(IdentityConfig.store).to receive(:state_tracking_enabled).and_return(false)
+        attach_and_submit_images
+
+        expect(DocAuthLog.find_by(user_id: @user.id).state).to be_nil
+      end
+    end
+
+    context 'standard mobile flow' do
+      it 'proceeds to the next page with valid info' do
+        perform_in_browser(:mobile) do
+          visit_idp_from_oidc_sp_with_ial2
+          sign_in_and_2fa_user(@user)
+          complete_doc_auth_steps_before_document_capture_step
+
+          expect(page).to have_current_path(idv_document_capture_url)
+          expect_step_indicator_current_step(t('step_indicator.flows.idv.verify_id'))
+          expect(page).not_to have_content(t('doc_auth.headings.document_capture_selfie'))
+
+          # doc auth is successful while liveness is not req'd
+          use_id_image('ial2_test_credential_no_liveness.yml')
+          submit_images
+
+          expect(page).to have_current_path(idv_ssn_url)
+          expect_costing_for_document
+          expect(DocAuthLog.find_by(user_id: @user.id).state).to eq('NY')
+
+          fill_out_ssn_form_ok
+          click_idv_continue
+          complete_verify_step
+          expect(page).to have_current_path(idv_phone_url)
+        end
+      end
+    end
+    context 'selfie check' do
+      let(:selfie_check_enabled) { true }
```

Comment:
it doesn't appear we're using this anywhere this is declared ... even outside of your tests ... let's scrap it thoughout this file?